plugins/dfu/dfu-device.c

```diff
@@ -1270,7 +1270,7 @@ dfu_device_probe (FuUsbDevice *device, GError **error)
 	/* hardware rom Jabra literally reboots if you try to retry a failed
 	 * write -- there's no way to avoid blocking the daemon like this... */
 	if (fu_device_has_custom_flag (FU_DEVICE (device), "attach-extra-reset"))
-		g_usleep (10 * G_USEC_PER_SEC);
+		fu_device_msleep_with_progress (FU_DEVICE (self), 10000);
```

Comment:
Maybe 10* usec_sec

Reply:
I called the method **m**sleep, as in miliseconds. Perhaps that's a mistake; perhaps it should be seconds -- we don't want to show the 0..100% progressbar for any less than one second anyway. Then it would be `fu_device_sleep_with_progress (FU_DEVICE (self), 10); /* seconds */`

Reply:
Sounds good to me

Reply:
Done